.github/workflows/ci.yml

```diff
@@ -73,6 +73,9 @@ jobs:
       # it comments on PRs to point out linter violations).
       - name: Lint
         uses: golangci/golangci-lint-action@v2
+        strategy:
+          matrix:
+            go-version: [1.17.x]
```

Comment:
It looks like [this](https://github.com/golangci/golangci-lint-action/issues/434) is the actual problem and this is the third solution I am seeing for the same problem :)
Did you consider those as well, asking to make sure  that we are aligned as much as possible.

- Skip go installation: https://github.com/crossplane/provider-azure/blob/afeda3e320643512c266f9bce161a50052d5c0f2/.github/workflows/ci.yml#L78
- Upgrade github action to v3: https://github.com/oam-dev/kubevela/pull/3571/files